src/ios/CDVFileTransfer.m

```diff
@@ -552,6 +552,8 @@ - (void)connectionDidFinishLoading:(NSURLConnection*)connection
 
     if (self.direction == CDV_TRANSFER_UPLOAD) {
         uploadResponse = [[NSString alloc] initWithData:self.responseData encoding:NSUTF8StringEncoding];
+        if (uploadResponse == nil)
+            uploadResponse = [[NSString alloc] initWithData: self.responseData encoding:NSISOLatin1StringEncoding];
```

Comment:
It does not look like there is a straightforward way to support this in Android, Windows. Perhaps we should make iOS itself consistent - in terms of handling both `upload` & `download` for this encoding. Document the limited support only in iOS, and file a `Feature Request` JIRA to add support in Windows, Android. Also, we need to add a test case. @codable  Would you be able to help with some of this?